authors.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+include_once "a_model.php";
+
+class Authors_model extends Model{
+
+    function __construct(){
+        $this->_table = "author";
+        parent::__construct();
+
+    }
+
+    function add_one($data_add) {
```

Comment:
Please make this method universal or more simple. Same comment for second model.